### PR TITLE
chore: update changelog for v0.33.0 release

### PR DESCRIPTION
**Pull request by gabrielmer**
# Description

Changelog for v0.33.0

Please review carefully and suggest highlights, upgrade instructions or notes that should be added :)

# Changes

- [x] updated changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
+## v0.33.0 (2024-09-25)
+
+#### Notes:
+
+* The `--pubsub-topic` CLI configuration has been deprecated and support for it will be removed on release v0.35.0. In order to migrate, please use the `--shard` configuration instead. For example, instead of `--pubsub-topic=/waku/2/rs/<CLUSTER_ID>/<SHARD_ID>`, use `--cluster-id=<CLUSTER_ID>` and `--shard=<SHARD_ID>`
```
Comment (SionoiS) [inline CHANGELOG.md:5]: Maybe add some words about `--cluster-id` should not be repeated?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+* The `--rest-private` CLI configuration is no longer supported. Please remove any reference to it when running your nodes
```
Comment (SionoiS) [inline CHANGELOG.md:6]: `no longer supported` Does that mean deprecated or removed?

**Review by SionoiS — APPROVED** [review]
Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (gabrielmer) [reply CHANGELOG.md:6]: It means removed! Will change the wording

**Review by richard-ramos — APPROVED** [review]

**Review by darshankabariya — APPROVED** [review]
LGTM

**Review by NagyZoltanPeter — APPROVED** [review]
Thanks for it!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,63 @@
...
+* a new experimental reliability protocol has been implemented, leveraging StoreV3 to confirm message delivery
+* (add highlight of the peer exchange rate limiting)
+
+### Bug Fixes
+
+- setting up node with modified config ([#3036](https://github.com/waku-org/nwaku/issues/3036)) ([8f289925](https://github.com/waku-org/nwaku/commit/8f289925))
+- get back health check for postgres legacy ([#3010](https://github.com/waku-org/nwaku/issues/3010)) ([5a0edff7](https://github.com/waku-org/nwaku/commit/5a0edff7))
+- libnegentropy integration ([#2996](https://github.com/waku-org/nwaku/issues/2996)) ([c3cb06ac](https://github.com/waku-org/nwaku/commit/c3cb06ac))
+- peer-exchange issue ([#2889](https://github.com/waku-org/nwaku/issues/2889)) ([43157102](https://github.com/waku-org/nwaku/commit/43157102))
+- copy libnegentropy.so from nim-build image ([#2991](https://github.com/waku-org/nwaku/issues/2991)) ([2edc0bc5](https://github.com/waku-org/nwaku/commit/2edc0bc5))
```
[the remark this answers is not in the document]
Comment (gabrielmer) [reply CHANGELOG.md:34]: ooh perfect, will remove it then so it doesn't add noise to the changelog. Thank you!

**Pull request merged**